runtime/compiler/optimizer/J9Simplifier.cpp

```diff
@@ -266,89 +245,167 @@ J9::Simplifier::foldAbs(TR::Node *node)
    return node;
    }
 
-TR::Node *
-J9::Simplifier::simplifyiCallMethods(TR::Node * node, TR::Block * block)
+template<typename T> struct BSWAP
```

Comment:
Why the template - can't we handle this case in a more traditional manner? Template handling in different compiles can be highly variable so I'd like to keep away from them if there isn't a good reason for them.